app/index.html

```diff
@@ -26,6 +28,8 @@
         <p class="l-text-center">&copy; 2015 &bull; <a href="https://github.com/mozilla/chronicle/" target="_blank">Mozilla</a></p>
       </div>
     </div>
-    <script data-main="assets/scripts/main" src="assets/bower_components/requirejs/require.js"></script>
+
+    <script data-main="scripts/main" src="assets/bower_components/requirejs/require.js"></script>
```

Comment:
You can replace this with the following if you run useminprepare after the requirejs task:

```
<!-- build:js scripts/compiled.js -->
<script data-main="scripts/main" src="assets/bower_components/requirejs/require.js"></script>
<!-- endbuild -->
```